AutomatedTesting/Gem/PythonTests/AWS/README.md

```diff
@@ -20,16 +20,18 @@
 3. Open a new CLI window at the engine root and set the following environment variables:
     * Windows
         ```
-            Set O3DE_AWS_PROJECT_NAME=AWSAUTO
             Set O3DE_AWS_DEPLOY_REGION=us-east-1
             Set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
+            Set BRANCH_NAME={your_branch_name}

```

Comment:
I'm not sure what "branch name" and "pipeline name" mean when I'm running this locally. Branch = git branch, I assume? But what should pipeline be?